crates/dht/src/arc_set.rs

```diff
@@ -114,6 +114,22 @@ impl ArcSet {
         self.inner.len()
     }
 
+    /// Create an [ArcSet] from a list of sectors indices.
```

Comment:
```suggestion
    /// Create an [ArcSet] from a list of sector indices.
```

Reply:
Code replaced per comment above about compressing the arc sets

Reply:
Which code is replaced? My suggestion was about the comment "a list of sectors indices".

Reply:
This was an `into_raw` and a `from_raw`, but is now an `encode` and `decode` that has different documentation